DependencyInjection/EkinoDataProtectionExtension.php

```diff
@@ -39,17 +46,50 @@ public function load(array $configs, ContainerBuilder $container)
         if (!$config['encrypt_logs']) {
             $container->removeDefinition('ekino_data_protection.monolog.processor.gdpr');
         }
+
+        if ($config['use_sonata_admin']) {
+            if (!class_exists(SonataAdminBundle::class)) {
+                throw new \RuntimeException('Please add ``sonata-project/admin-bundle`` package and load the bundle in your AppKernel');
+            }
+
+            $this->configureLogsAdminController($container);
+            $this->configureLogsAdmin($container);
+        }
     }
 
     /**
      * @param array            $config
      * @param ContainerBuilder $container
      */
-    private function configureEncryptor(array $config, ContainerBuilder $container)
+    private function configureEncryptor(array $config, ContainerBuilder $container): void
     {
         $container
             ->findDefinition('ekino_data_protection.encryptor')
             ->replaceArgument(0, $config['method'])
             ->replaceArgument(1, $config['secret']);
     }
+
+    /**
+     * @param ContainerBuilder $container
+     */
+    private function configureLogsAdmin(ContainerBuilder $container): void
+    {
+        $definition = new Definition(LogsAdmin::class, [
```

Comment:
Should be declared in `sonata_admin.xml` loaded only if `use_sonata_admin` is `true`